clustering/kmeans_sample.ipynb

```diff
@@ -0,0 +1,518 @@
+{
```

Comment:
<ul><li>Will you elaborate these methods more detailed in this notebook?</li><li><code>Gap statistic method</code> is given but no introduction, code, or even reference?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Women-in-Data-Science-Taipei/2020-WiDS-Taipei-MLCC-Workshop/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>